fastlane.js

```diff
@@ -12,116 +11,124 @@ const teamupToken = process.env.HUBOT_TEAMUP_TOKEN
+
+const getNecessaryNode = (response)=>{
+    const node = response.data.data.organization.projectV2.items;
+    const issueList = [];
+
+    node.nodes.forEach(value => {
+        const title = value.content.title;
+        const url = value.content.url;
+        const nameNode = value.fieldValues.nodes.find(field => field.name);
+        const name = nameNode.name
+
+        if (title && url && name && name === `${fastLaneColumnName}`) {
+            issueList.push([title,url])
+        }
     })
-    robot.http(`https://api.github.com/orgs/${organisation}/projects`)
-      .headers({Accept: 'application/json', Authorization: `Basic ${auth}`})
-      .get()((err, response, body) => {
-          if (err) {
-            robot.emit('error', `problem getting projects list: '${err}'`)
-            return
-          }
-          let parsedData = {}
-          try {
-            parsedData = JSON.parse(body)
-          } catch (e) {
-            robot.emit('error', `problem parsing '${body}' as JSON`)
-            return
-          }
 
-          if (!Array.isArray(parsedData)) {
-            robot.emit('error', `Response body cannot be parsed to an array. Content: "${body}"`)
-            return
+    return [node,issueList]
+}
+
+const generateQuery = (item) => {
+    return `query {
+    organization(login: "${organisation}") {
+    projectV2(number: 386) {
+      items(${item}) {
+        totalCount
+        nodes {
+          content {
+            ... on Issue {
+              title
+              url
+            }
+            ... on PullRequest {
+              title
+              url
+            }
+          }
+          fieldValues(last: 3) {
+            nodes {
+              ... on ProjectV2ItemFieldSingleSelectValue {
+                name
+              }
+              ... on ProjectV2ItemFieldTextValue {
+                text
+              }
+            }
           }
+        }
+        pageInfo {
+          endCursor
+          hasNextPage
+        }
+      }
+    }
+  }
+}`
+}
 
-          const data = parsedData.find(function (project) {
-            return project.name === projectName
-          })
+let endCursor = null
+let hasNextPage = true
+let cards = []
 
-          if (typeof data !== 'object' || typeof data.columns_url !== 'string' ) {
-            robot.emit('error', `could not find project '${projectName}' do you have the right permissions?`)
+module.exports = robot =>
+    setInterval(() => {
+        robot.error(function (err, res) {
+            robot.logger.error(err)
+            robot.send({room: room}, `there is an issue with the fastlane bot '${err}'`)
+        })
+        const sendRequest = () => {
+            if (hasNextPage) {
+                robot.http(GITHUB_GRAPHQL_API_URL)
+                    .headers({Accept: 'application/json', Authorization: `Basic ${auth}`})
+                    .data({query: generateQuery(`first: 30, after: "${endCursor}"`)})
+                    .get()((err, response) => {
```

Comment:
will `get` work, I think it requires `post`
```suggestion
                    .post()((err, response) => {
```